src/bosh-director/lib/bosh/director/ip_util.rb

```diff
@@ -50,19 +52,19 @@ def process_range(range)
       parts = range.split("-")
       parts.each { |part| part.strip! }
       if parts.size == 1
-        range = NetAddr::CIDR.create(parts[0])
-        first_ip = range.first(:Objectify => true).to_i
-        last_ip = range.last(:Objectify => true).to_i
+        range = NetAddr::IPv4Net.parse(parts[0])
+        first_ip = range.nth(0).addr
```

Comment:
Guessing that range is usually something like 10.0.0.1-10.10.0.1 but in this case it is simply 10.0.0.1. Why don't we calculate first and last ip similar to the "parts.size == 2"-case
`
first_ip = NetAddr::IPv4Net.parse(parts[0])
last_ip = NetAddr::IPv4Net.parse(parts[1])
`